ghcide/src/Development/IDE/GHC/Compat/Core.hs

```diff
@@ -121,9 +124,12 @@ module Development.IDE.GHC.Compat.Core (
       TyCoRep.CoercionTy
       ),
     pattern FunTy,
+    pattern ConPatIn,
     Development.IDE.GHC.Compat.Core.splitForAllTyCoVars,
+#if !MIN_VERSION_ghc(9,0,0)
     Development.IDE.GHC.Compat.Core.mkVisFunTys,
     Development.IDE.GHC.Compat.Core.mkInfForAllTys,
+#endif
```

Comment:
Did this cause any issues or something? I would prefer to have as few CPP statements as possible, as you lose very quickly the big picture about what happens in Core.hs

Reply:
Oh, I see. Yeah, that should be fairly easy to workaround. I was trying to avoid having duplicate
```
foo = Module.foo
```
definitions and just reexporting the existing things. I think removing these pragmas would just cause a few "duplicate exports" warnings, so another option could be to just disable that warning in this file.